plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/utils.ts

```diff
@@ -19,7 +19,9 @@ export const addIsAddedClassToPatternPreview = (
 	blocks: BlockInstance[]
 ) => {
 	patternListEl.querySelectorAll( '.is-added' ).forEach( ( element ) => {
-		element.classList.remove( 'is-added' );
+		const htmlElement = element as HTMLElement;
+		htmlElement.blur();
```

Comment:
Do we need to `blur` it? It's a bit problematic for keyboard navigation: you focus one pattern, press <kbd>Space</kbd> to add and then you can't press <kbd>Space</kbd> again to remove it as the focus was lost. It also makes it harder navigating between patterns after adding one.

Reply:
This aspect of the behavior can be improved and was one of the main reasons its currently blocked. Not only does the issue you've highlighted exist but there's no differentiating UX between "pattern is selected/focused" and "pattern has been inserted".